src/signed_data.rs

```diff
@@ -373,6 +373,25 @@ const ED_25519: AlgorithmIdentifier = AlgorithmIdentifier {
     asn1_id_value: untrusted::Input::from(include_bytes!("data/alg-ed25519.der")),
 };
 
+/// All signature algorithms supported by the webpki crate.
+///
+/// Note the list depends on whether `alloc` feature is on or off.
+pub static ALL_SIGNATURE_ALGORITHMS: &[&SignatureAlgorithm] = &[
```

Comment:
Sorry, I know I asked you to move this into the library but that was a mistake on my part. Please move it back to where you had it before and add a comment to this:

// Keep in sync with `ALL_SIGALGS` in tests/integration.rs.